src/rtHttpCache.cpp

```diff
@@ -222,7 +222,7 @@ rtData& rtHttpCacheData::contentsData()
   return mData;
 }
 
-rtError rtHttpCacheData::data(rtData& data)
+rtError rtHttpCacheData::data(rtData& data, bool readCachedDataInChunks)
```

Comment:
Let's not modify the data() API or call.  Instead, let's create an API on rtHttpCacheData that will setup it up to be read lazily.